packages/ffe-cards/less/detail-list-card.less

```diff
@@ -0,0 +1,54 @@
+.ffe-detail-list-card {
+    background-color: var(--ffe-v-detail-list-card-background-color);
+    border-radius: var(--ffe-v-cards-common-card-border-radius);
+    font-family: var(--ffe-g-font);
+    font-variant-numeric: tabular-nums;
+
+    &__item:not(:last-child) {
+        border-bottom: 1px solid var(--ffe-v-detail-list-card-border-color);
+    }
+
+    &__item {
+        padding: var(--ffe-spacing-sm);
+        display: grid;
+        grid-template-columns: 1fr 2fr;
+        justify-content: space-between;
+        gap: 8px;
+        word-break: break-all;
+        text-wrap: balance;
+
+        &:is(:first-child) {
+            border-radius: var(--ffe-v-cards-common-card-border-radius)
+                var(--ffe-v-cards-common-card-border-radius) 0 0;
+        }
+
+        &:is(:last-child) {
+            border-radius: 0 0 var(--ffe-v-cards-common-card-border-radius)
+                var(--ffe-v-cards-common-card-border-radius);
+        }
+
+        &-label {
+            font-family: var(--ffe-g-font-heading-small);
+            color: var(--ffe-g-secondary-color);
+        }
+
+        &-value {
+            text-align: right;
+        }
+
+        &--clickable {
+            cursor: pointer;
+
+            .ffe-detail-list-card__item-label {
+                color: var(--ffe-g-text-color);
+            }
+
+            &:hover,
```

Comment:
Tror vi burde ta en runda med og skjula hover på device men man kan ju begynne her om man ønsker allrede: Provade selv på emualtorn og så ut att funka. 
https://stackoverflow.com/questions/23885255/how-to-remove-ignore-hover-css-style-on-touch-devices/64553121#64553121